pkg/generate/app/dockerimagelookup.go

```diff
@@ -76,7 +76,7 @@ func (r DockerClientSearcher) Search(terms ...string) (ComponentMatches, error)
 				termMatches = append(termMatches, matches...)
 			case ErrNoMatch:
 			default:
-				return nil, err
+				glog.V(1).Infof("An error occurred searching the Docker registry: %v", err)
```

Comment:
The registry is just one of our search sources. If we can't find anything we should simply move on to search on the next source (the local docker daemon), if available. This is basically doing the same for the registry as we do for other sources. If for example we don't have a local docker accessible, we simply don't search it or get any results from it. Same for the openshift server if we can't connect to it to search for image streams or templates.